.travis.yml

```diff
@@ -21,6 +22,8 @@ before_script:
 
   # Install composer dependencies
   - composer validate
+  - composer require --no-update silverstripe-themes/simple '~3.2'
+  -  echo -e "SilverStripe\View\SSViewer:\n  themes:\n    - '\$public'\n    - 'simple'\n    - '\$default'" > mysite/_config/theme.yml
```

Comment:
Can't we just use the silverstripe/installer for this?

Reply:
Maybe. I was confused about how the recipes mix together in such circumstance, since this is a recipe that already includes various bits and bobs that stop other recipe dependencies from overwriting...

Reply:
Give it a crack I reckon

Reply:
Still got more issues to solve, so will do :)

Reply:
Oddly, it passes like this, but fails if I try with installer instead.
It seems `installer` will _not_ implant `theme.yml`, despite it being listed in `project-files`, much the same way that `recipe-core` does with `mysite.yml` (which _is_ extant).
Not sure what's going on there, but this seems much more reliable, at least in the short term.

Reply:
You’ll probably need to switch the phpunit config from framework to cms bootstrapping

Reply:
False alert, I must have had a contaminated project dir when trying installer before. Good call on the bootstrap, I'll try switching that too.